### PR TITLE
maps/assets: Fix select2 for polygon areasetting

### PR DESCRIPTION
**Pull request by rmader**
`select2` is a jQuery extension, thus we need to use the window-global
jQuery instance instead of the webpack provided one.

Depends on https://github.com/liqd/adhocracy4/pull/673

### CODE REVIEW

**Comment by rmader** [comment]
@phillimorland got a much better solution!

File: package.json
Comment (fuzzylogic2000) [inline package.json:23]: Ah, that's where the wrong versions came from yesterday. :) We left the hash out of depending PRs and only added it after the a4 PRs are merged. Then, people had to locally try it by either adding the a4 hash or using the local a4 and there are no wrong hashes introduced by accident.

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! Will merge after another a4-Pr is merged and update a4 in here.

**Pull request merged**
